z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -34,7 +34,7 @@ The playbook contains the following topics:
 
   3. Requirements pre-check before the installation
 
-**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11for KVM.
+**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5, 1.1.6 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11 for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11 for KVM.
```

Comment:
Currently we support ocp 4.10, 4.11 and 4.12. Please help update it.

Reply:
done.